src/py21cmfast/inputs.py

```diff
@@ -626,6 +626,19 @@ class FlagOptions(StructWithDefaults):
         Determines whether to use a fixed vcb=VAVG (*regardless* of USE_RELATIVE_VELOCITIES). It includes the average effect of velocities but not its fluctuations. See Muñoz+21 (2110.13919).
     USE_VELS_AUX: bool, optional
         Auxiliary variable (not input) to check if minihaloes are being used without relative velocities and complain
+    USE_Radio_ACG: bool, optional
+        Determines whether to use radio excess background from ACG, set this to True and AstroParams.fR > 0 to activate
+    USE_Radio_MCG: bool, optional
+        Determines whether to use radio excess background from MCG, set this to True and AstroParams.fR_mini > 0 to activate
+    USE_Radio_PBH: bool, optional
+        Determines whether to use radio excess background from PBH, set this to True and AstroParams.log10_fbh > -200 to activate
+    USE_Hawking_Radiation: bool, optional, see 2108.13256
```

Comment:
For visual consistency with other parameters, can we make these all caps? I actually don't like the all-caps myself, but consistency is better than opinion ;-)

Reply:
Thanks Steven, all the new FlagOptions are now in caps